packages/legend-studio/src/components/editor/edit-panel/mapping-editor/MappingTestEditor.tsx

```diff
@@ -687,17 +697,42 @@ export const MappingTestEditor = observer(
             paper: 'editor-modal__content',
           }}
         >
-          <div className="modal modal--dark editor-modal execution-plan-viewer">
+          {/* <ExecutionNodesViewer /> */}
+          <div className="modal modal--dark editor-modal">
             <div className="modal__header">
               <div className="modal__title">Execution Plan</div>
             </div>
             <div className="modal__body">
-              <TextInputEditor
-                inputValue={planText}
-                isReadOnly={true}
-                language={EDITOR_LANGUAGE.JSON}
-                showMiniMap={true}
-              />
+              {planMeta instanceof ExecutionPlan && (
+                <SplitPane
```

Comment:
Might make sense to move this block out to the execution-plan-viewer folder as a component